provider/helm/common.go

```diff
@@ -17,17 +17,21 @@ var ErrKeelConfigNotFound = errors.New("keel configuration not found")
 
```

Comment:
@rusenask this change broke polling for me, i was a bit slow on the upgrades recently

not sure why its broken but in the debug logs no versions a logged

I wonder if you have assumed that SemVer _must_ have 3 points when 2 points is perfectly valid e.g. 12.4 and not 12.4.0